File: bot.js
const Discord = require('discord.js');
const client = new Discord.Client();

client.once('ready', () => {
	console.log('Ready!');
});

if (message.content === '!ping') {
	// send back "Pong." to the channel the message was sent in
	message.channel.send('Pong!');
}

client.login('TOKEN');

File: bot.py
# import these :)
import discord
from discord.ext import commands

# Add prefix (needed) and a description! (optional)
bot = commands.Bot(command_prefix="{PREFIX}", description="{DESCRIPTION}")

# When the bot is ready...
@bot.event()
async def on_ready(ctx):
  print('Ready!')
 
# Here is how you do a responder command!
@bot.command()
async def ping(ctx)
  await ctx.channel.send('Pong!')
  
bot.run('TOKEN')
